fuzz/src/main/java/edu/berkeley/cs/jqf/fuzz/ei/ZestGuidance.java

```diff
@@ -238,7 +238,7 @@
     static final double MEAN_MUTATION_SIZE = 4.0; // Bytes
 
     /** Whether to save inputs that only add new coverage bits (but no new responsibilities). */
-    static final boolean SAVE_NEW_COUNTS = true;
+    static final boolean SAVE_NEW_COUNTS = Boolean.getBoolean("jqf.ei.SAVE_NEW_COUNTS");
```

Comment:
So this makes SAVE_NEW_COUNTS `false` by default if the option is not given, which severely breaks backward compatibility. Users running new fuzzing sessions will see completely different coverage. Maybe flip this to an option `jqf.ei.DISABLE_SAVE_NEW_COUNTS` so that the default behavior is maintained when no option is given?

Reply:
right, my bad